app/src/main/java/com/bignerdranch/android/criminalintent/DatePickerFragment.kt

```diff
@@ -0,0 +1,58 @@
+package com.bignerdranch.android.criminalintent
+
+import android.app.DatePickerDialog
+import android.app.Dialog
+import android.os.Bundle
+import android.widget.DatePicker
+import androidx.fragment.app.DialogFragment
+import java.util.Calendar
+import java.util.Date
+import java.util.GregorianCalendar
+
+private const val ARG_DATE = "date"
```

Comment:
it would be better to store this one in the `companion object` of `DatePickerFragment`.

Reply:
Done!